src/modules/notes/notes.service.ts

```diff
@@ -465,4 +462,8 @@ export class NotesService {
   async remove(id: string) {
     return this.prisma.note.delete({ where: { id }, include });
   }
+
+  getCount(accessFilters: Prisma.NoteWhereInput) {
```

Comment:
```suggestion
  getCount(filters: Prisma.NoteWhereInput) {
```

These are not necessarily ACCESS filters, are they?